functions/superLikeRedirect.js

```diff
@@ -31,6 +32,7 @@ function apiGetSuperLikeInfo(id) {
 const app = express();
 app.use(helmet());
 app.use(cookieParser());
+app.use(express.static(path.join(__dirname, '../public_base')));
```

Comment:
We don't need `express.static` since files in `public_base` is directly served by Firebase hosting
Also note that any files in `public_base` also appears in `button.like.co`

Reply:
Seems not working, maybe need to update firebase.json?

Reply:
need to change the ignore rule to accept .wellknown i guess
https://github.com/likecoin/likecoin-button/blob/master/firebase.json#L37

Reply:
If so, what is the robot.txt in the public_base for?

Reply:
robot does not begin with a .

Reply:
https://s.like.co/robots.txt

Reply:
Then why isn't work on https://s.rinkeby.like.co/robot.txt

Reply:
https://s.rinkeby.like.co/robots.txt

Reply:
on other hand https://s.rinkeby.like.co/.gitkeep won't be uploaded, since `**/.*`

Reply:
I have a problem, the https://s.rinkeby.like.co/.well-known/apple-app-site-association some how return empty

Reply:
Made modification on `firebase.json` according to https://gist.github.com/mbleigh/c9140c97a342b9d2e7cd727e21b6268f#file-firebase-json-L2 , should work